atom/browser/api/atom_api_tray.cc

```diff
@@ -210,7 +210,7 @@ void Tray::PopUpContextMenu(mate::Arguments* args) {
 
 void Tray::SetContextMenu(v8::Isolate* isolate, mate::Handle<Menu> menu) {
   menu_.Reset(isolate, menu.ToV8());
-  tray_icon_->SetContextMenu(menu->model());
+  tray_icon_->SetContextMenu(menu.IsEmpty() ? nullptr : menu->model());
```

Comment:
Can `menu->model()` just return a `nullptr` when it's empty?

Reply:
menu itself is empty reference

Reply:
Oh, right.